src/app/executions/tasks-runtime/tasks-runtime.component.html

```diff
@@ -24,13 +24,16 @@
 
       <td [ngbPopover]="runtimeTooltip" container="body" triggers="mouseenter:mouseleave">
         <div class="stackedBarChart">
-          <ng-template #runtimeTooltip>Started: {{task.created_at}}<br/>
-            Updated: {{task.updated_at}}<br/>
-            Percent: {{task.duration_sec === 0 ? '< ' : ''}}{{task.barWidth | number}}%</ng-template>
+          <ng-template #runtimeTooltip>Started: {{task.started_at}}<br/>
+            Finished: {{task.finished_at}}<br/>
+            Percent: {{task.duration_sec === 0 ? '< ' : ''}}{{task.runningBarWidth | number}}%<br/>
+            Total: {{task.duration_sec === 0 ? '< ' : ''}}{{task.totalBarWidth | number}}%</ng-template>
 
           <div class="inner">
             <div class="prehighlight" [style.width.%]="task.preBarWidth"></div>
-            <div class="highlight" [style.width.%]="task.barWidth"></div>
+            <div class="waiting-highlight" [style.width.%]="task.waitingBeforeBarWidth"></div>
```

Comment:
See comment below - we can save some css code by setting 2 classes on this item `classes="waiting highlight"`. (Same for line 36)

Reply:
done